submissions/HTML-forms/Contact form page.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"content="width=device-width,initial-scale=1.0">
+    <title>Contact Form Page</title>
+</head>    
+<body>
+    <div class="topnav">
+                    <a class="active" href="#home">Home</a>
+                    <a href="#category">Category</a>
+                    <a href="file:///C:/Users/halim/OneDrive/Desktop/Dufuna-Fem/submissions/HTML/index.html">Product</a>
```

Comment:
Your link is too long. Since the page you are redirecting to is in the same folder, you can simply do ../HTML/index.html